$MirOS: ports/audio/lame/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:42:54 tg Exp $
$OpenBSD: patch-configure,v 1.1 2001/08/26 12:00:38 espie Exp $
--- configure.in.orig	Sun Jul 25 14:51:50 2004
+++ configure.in	Wed Dec 28 17:38:47 2005
@@ -49,7 +49,7 @@ LIB_MAJOR_VERSION=0
 
 # increase this when changes are made, but they are upward compatible
 # to previous versions
-LIB_MINOR_VERSION=0
+LIB_MINOR_VERSION=1
 
 dnl # work around for a bug, don't know where it is exactly
 if test "${ac_cv_cygwin}" = "yes"; then
